changelog/2022-02-28.md

```diff
@@ -0,0 +1,17 @@
+## :wrench: February 28, 2022
+
+**CLI version: 0.8.0**
+
+Spring cleaning came early. This release features a lot of backend magic that improves overall stability and UX with Grid. 
+
+Upgrade your CLI to `v0.8.0` with `pip install lightning-grid --upgrade`  
+
+Find us in our [Slack Community](http://gridai-community.slack.com) to say hi and/or to express your thoughts/questions.
+
+### Notable Fixes and Enhancements
```

Comment:
We probably also want to add the removal of unhelpful stacktrace error messages.

Reply:
this was not in the list that @ematta provided me. was this added to the LTS branch? and has it been tested in QA?